ui/pages/send/send-content/send-hex-data-row/send-hex-data-row.test.js

```diff
@@ -0,0 +1,66 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { ASSET_TYPES } from '../../../../../shared/constants/transaction';
+import { renderWithProvider } from '../../../../../test/jest/rendering';
+import SendHexDataRow from './send-hex-data-row';
+
+const middleware = [thunk];
+const renderComponent = () => {
+  const store = configureMockStore(middleware)({
+    metamask: { identities: [], provider: {} },
+    send: {
+      userInputHexData: {
+        input: '',
+        error: null,
+      },
+      asset: {
+        type: ASSET_TYPES.NATIVE,
+      },
+    },
+  });
+  return renderWithProvider(<SendHexDataRow />, store);
+};
```

Comment:
actual store object can also be used for testing and will simulate actual store: https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-popover.test.js#L32

Reply:
Fixed [here](https://github.com/MetaMask/metamask-extension/pull/14724/commits/18a653d89ebcb57a0d9280c36b782cb827dcdfe7)